### PR TITLE
docs : Updates a line in README.md

### LINKED ISSUE #820: Remove the Python versions contradiction in README file
## Description
As a contributor,
I need no contradictions in the README file,
so that I won't be doubtful of the dependencies.

## Mocks
image
Both the images (above and below) contradicts with the statement `Python3` and `Python3.6`
image

### Update [Required]
- [ ] Update `Python 3.6` statement to `Python 3` 

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.1 hour

### PR DESCRIPTION
**Pull request by Surajv311**
### Description

Fixes #820

### Type of Change:

- Documentation

### Checklist:

- [X] My PR follows the style guidelines of this project
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by PrashanthPuneriya — APPROVED** [review]
LGTM :+1:

**Review by PrashanthPuneriya — CHANGES_REQUESTED** [review]
Can you please update the commit message so that you follow the [guidelines](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide) and also please follow the PR [guidelines](https://github.com/anitab-org/mentorship-backend/blob/develop/.github/PULL_REQUEST_TEMPLATE.md) as well :)

**Comment by Surajv311** [comment]
@PrashanthPuneriya  Ok.

**Comment by Surajv311** [comment]
@PrashanthPuneriya Could you please review my PR ?

**Review by vj-codes — CHANGES_REQUESTED** [review]
@Surajv311 congrats on your first contribution to the repo 🎉 
Please update the PR title ,similar to to commit message, then you're done!

**Comment by Surajv311** [comment]
@vj-codes . I have made the required changes. Could you please review ?

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system)
 [![Python 3.6](https://img.shields.io/badge/python-3.6-blue.svg)](https://www.python.org/downloads/release/python-360/)
 
-
+
```
Comment (PrashanthPuneriya) [inline README.md]: Can you please remove this extra space?

**Review by vj-codes — DISMISSED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (Surajv311) [reply README.md]: @PrashanthPuneriya Done! Could you please review ?

File: README.md [hunk 2]
```diff
@@ -5,7 +5,6 @@
 [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system)
 [![Python 3.6](https://img.shields.io/badge/python-3.6-blue.svg)](https://www.python.org/downloads/release/python-360/)
 
-
```
Comment (PrashanthPuneriya) [inline README.md]: I think removing this line doesn't make any difference. But, there is no need of removing this line. Fix this and you are good :)

**Comment by vj-codes** [comment]
@PrashanthPuneriya hey the badge in readme should be changed subsequently. Will it be in the same issue or another one?

**Review by SanketDG — CHANGES_REQUESTED** [review]
Please squash the commits into one and write a proper description in the message.

"Update a line" does not say anything useful, "Use Python 3 instead of specific version in README" is useful enough :)

**Comment by Surajv311** [comment]
@SanketDG @PrashanthPuneriya . I have made the required changes. Could you please give a review ?

**Comment by PrashanthPuneriya** [comment]
> @PrashanthPuneriya hey the badge in readme should be changed subsequently. Will it be in the same issue or another one?

Ahh nice catch @vj-codes ! There are lot of newcomers willing to contribute to this project. Though, the badge can be changed here subsequently I think it's better if we create a new issue so that everyone can get a fair chance. Please feel free to create one @vj-codes

**Comment by vj-codes** [comment]
yes sure @PrashanthPuneriya

**Review by PrashanthPuneriya — APPROVED** [review]

**Review by SanketDG — APPROVED** [review]
Great job! Looks good to merge!

**Review by vj-codes — APPROVED** [review]
LGTM 🎉

**Review by isabelcosta — APPROVED** [review]
Great contribution @Surajv311 🎉 💪

**Pull request merged**